packages/ui/src/theme/__tests__/defaultTheme.test.ts

```diff
@@ -528,24 +528,24 @@ describe('@aws-amplify/ui', () => {
         --amplify-components-sliderfield-large-thumb-height: 1.5rem;
         --amplify-components-stepperfield-flex-direction: column;
         --amplify-components-stepperfield-input-text-align: center;
-        --amplify-components-switchfield-disabled-opacity: var(--amplify-opacities-60);
-        --amplify-components-switchfield-focused-shadow: var(--amplify-shadows-small);
-        --amplify-components-switchfield-large-font-size: var(--amplify-font-sizes-large);
-        --amplify-components-switchfield-small-font-size: var(--amplify-font-sizes-small);
         --amplify-components-switchfield-label-padding: var(--amplify-space-xs);
         --amplify-components-switchfield-thumb-background-color: var(--amplify-colors-background-primary);
         --amplify-components-switchfield-thumb-border-color: var(--amplify-colors-border-tertiary);
         --amplify-components-switchfield-thumb-border-radius: var(--amplify-radii-xxxl);
-        --amplify-components-switchfield-thumb-checked-transform: var(--amplify-transforms-slide-x-medium);
         --amplify-components-switchfield-thumb-transition-duration: var(--amplify-time-medium);
         --amplify-components-switchfield-thumb-width: var(--amplify-space-relative-medium);
         --amplify-components-switchfield-track-background-color: var(--amplify-colors-background-tertiary);
         --amplify-components-switchfield-track-border-radius: var(--amplify-radii-xxxl);
-        --amplify-components-switchfield-track-checked-background-color: var(--amplify-colors-brand-primary-60);
```

Comment:
We need to be careful of breaking changes like this. If there are customers using this variable SwitchField styling will break. Can we link this one to the new variable for backwards compatibility?

Reply:
Same comment for these as well:
```
--amplify-components-switchfield-thumb-checked-transform
--amplify-components-switchfield-focused-shadow:
```

Reply:
Updated to revert these breaking token name changes.